test/unit/ngsiv2/plugins/event-plugin_test.js

```diff
@@ -0,0 +1,121 @@
+/*
```

Comment:
I have seen in the code part of the PR modifications regarding Alias Attribute Plugin and Expressions Plugin but not related with Events plugin. Is this correct (maybe I have miss these modifications)?

In this case, I understands the modifications with regards Event plugin in this PR are the inclusion of a new NGSIv2-oriented test in order to check the functionally is still working, isn't it?

Reply:
Yes, you are right, I haven't modified anything to support the functionality of Events plugin with NGSIv2, just added tests to demontrate that it works.

The reason is that this plugin just consists of the following lines:
```
function addEvents() {
    return (new Date()).toISOString();
}

exports.update = pluginUtils.createUpdateFilter(addEvents, config.getConfig().eventType || 'Event');
exports.query = pluginUtils.identityFilter;
```
And supporting NGSIv2 in pluginUtils was already addressed in the previous PR.

Reply:
Ok. NTC.